src/test/a11y/tests/media-routes.test.ts

```diff
@@ -122,5 +122,39 @@ describe('Accessibility - Media User Routes', () => {
                 testAccessibility(url, '', true, { 'search-input': 'Invalid case number' });
             });
         });
+
+        describe('Bulk Unsubscribe Confirmation Page', () => {
+            const url = '/bulk-unsubscribe-confirmation';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
+            });
+        });
+
+        describe('Pending Subscriptions Page', () => {
+            const url = '/pending-subscriptions';
+
+            beforeEach(() => {
+                sinon.restore();
+                sinon.stub(PublicationService.prototype, 'getCaseByCaseNumber').resolves(null);
+                sinon.stub(PublicationService.prototype, 'getCaseByCaseUrn').resolves(null);
+            });
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'search-input': '' });
+            });
+
+            describe('with invalid input data', () => {
+                testAccessibility(url, '', true, { 'search-input': 'Invalid case number' });
+            });
+        });
+
+        describe('Subscription Add Page', () => {
+            const url = '/subscription-add';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
```

Comment:
'input-autocomplete' is not the right post body for this test. Please check the controller code